docker/etc/pacman/mirrorlist.latest

```diff
@@ -0,0 +1,2 @@
+# Intentionally use the latest mirror list so that we can pick up the latest public keys from the core arch linux maintainers.
+Server=https://archive.archlinux.org/repos/last/$repo/os/$arch
```

Comment:
Oh I see this is what you were adding in. I guess this seems ok?

Reply:
Basically what is happening to our system [is this](https://nanxiao.me/en/update-keyring-first-if-your-arch-linux-is-old-enough/). The public keys on the system have expired and need to be updated prior to installing any more packages. This server, and the `pacman.latest.conf` file, enables the `install-package` script to update the keys prior to installing a package from our date locked version of Arch Linux.